compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/window/Popup.skiko.kt

```diff
@@ -85,23 +95,20 @@ fun Popup(
     onPreviewKeyEvent: ((KeyEvent) -> Boolean) = { false },
     onKeyEvent: ((KeyEvent) -> Boolean) = { false },
     content: @Composable () -> Unit
-) {
-    val popupPositioner = remember(alignment, offset) {
-        AlignmentOffsetPositionProvider(
-            alignment,
-            offset
-        )
-    }
-
-    Popup(
-        popupPositionProvider = popupPositioner,
-        onDismissRequest = onDismissRequest,
-        onKeyEvent = onKeyEvent,
-        onPreviewKeyEvent = onPreviewKeyEvent,
+) = Popup(
+    alignment = alignment,
+    offset = offset,
+    onDismissRequest = onDismissRequest,
+    properties = PopupProperties(
         focusable = focusable,
-        content = content
-    )
-}
+        dismissOnBackPress = true,
+        dismissOnClickOutside = focusable
```

Comment:
Why is this tied to `focusable`? A non-focusable popup should not have it's `onDismissRequest` called on outside click?

Reply:
This was the previous behaviour of this overload. It's now just explicit

Reply:
We're discussed it with you a month ago - https://github.com/JetBrains/compose-multiplatform-core/pull/611#discussion_r1248090480

Reply:
We discussed it with you a month ago - https://github.com/JetBrains/compose-multiplatform-core/pull/611#discussion_r1248090480